plugins/scripting/scripting-compiler/src/org/jetbrains/kotlin/scripting/compiler/plugin/impl/jvmCompilationUtil.kt

```diff
@@ -71,6 +71,19 @@ inline fun <T> withMessageCollector(
     }
 }
 
+inline fun <T> withMessageExtendedCollector(
```

Comment:
And here too, it seems that we don't need non-extended variant in scripting, so you can just update the `withMessageCollector` function instead.